### PR TITLE
Iz notebooks

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

**Comment by bradtaylor** [comment]
Qu'est-ce que c'est?

**Comment by bradtaylor** [comment]
Oh, it's a PR. Could you connect it with an issue? It then travels along with the card in the backlog, which is hecka nifty

**Comment by bradtaylor** [comment]
Connecting to "Basic cluster management"

File: src/libs/nav.js [hunk 1]
```diff
@@ -48,32 +65,59 @@ const findMatches = function(windowHash, checkingRedirects) {
...
+ * @param args
+ * @returns {string}
+ */
 export const isCurrentPath = function(k, ...args) {
   return getPath.apply(this, Array.from(arguments))
 }
 
+/**
+ * @param windowHash
+ * @returns {boolean} true
```
Comment (dmohs) [inline src/libs/nav.js]: "True if a redirect was executed?"

**Review by dmohs — APPROVED** [review]
👍

File: src/pages/workspaces/workspace/Container.js [hunk 2]
```diff
@@ -3,64 +3,74 @@ import { Component, Fragment } from 'react'
...
   return icon(shape, { size: 22, style: { opacity: 0.65, paddingRight: '1rem' } })
 }
 
+const tabComponents = {
+  dashboard: WorkspaceDashboard,
+  notebooks: WorkspaceNotebooks,
+  data: WorkspaceData,
+  jobs: () => div('when we get to it'),
+  history: () => div('okay, this is _data_ history, whatever that means'),
+  tools: () => div('These are totally Not Method Configs')
```
Comment (dmohs) [inline src/pages/workspaces/workspace/Container.js:36]: People are looking at this UI. I'm inclined to think we should have nice-looking placeholders here (sans snark ;) or remove the tabs until we've implemented something.

**Pull request merged**